backend/src/main/resources/liquibase/db-changelog-1.0.xml

```diff
@@ -266,7 +266,7 @@
                   referencedColumnNames="id"/>
             </column>
             <column name="part_number" type="int8">
-                <constraints nullable="false"/>
+                <constraints primaryKey="true" nullable="false"/>
```

Comment:
Move this to the PR doing liquibase changes because it seems like the original one was buggy.

Reply:
moved